### ISSUE TITLE
Ic2 Classic Retextures?

### ISSUE THREAD

**Issue by DrParadox7** [CyanideX/Unity#181]
Now that IC2 Classic has been updated to the latest 1.12 version are there still plans for its re-texture as there was already some progress since 2016 or has it since been abandoned? I can't seem to find a way to tell if it is in development due to only IC2 showing up but not specifying whether it is Experimental, Classic or both.

**Comment by CyanideX** [comment]
I guess I should reopen this issue: https://github.com/CyanideX/Unity/issues/55

**Comment by CindyLouWho03** [comment]
@DrParadox7 First of all is this the IC2Classic by not the IC2 team or is the the offical IC2 with the classic profile in the config. If the latter then you can use the new textures with the older recipes and code and then you can overite the textures with unitys from there

**Comment by CyanideX** [comment]
If you quickly Google "IC2 Classic" you will see it's the version by Speiger that was recently updated to 1.12. 😛

**Comment by InboundBark** [comment]
Is this still being worked on?

**Comment by CyanideX** [comment]
No, it is not currently being worked on but I may come back to this when I have some free time.
